src/Client.php

```diff
@@ -0,0 +1,221 @@
+     * Client constructor.
+     *
+     * @param \Badoo\Jira\REST\Client $RESTClient
+     */
+    public function __construct(\Badoo\Jira\REST\Client $RESTClient)
+    {
+        $this->RESTClient = $RESTClient;
+    }
+
+    /**
+     * @param string $issue_key
+     *
+     * @return \Badoo\Jira\Issue
+     *
+     * @throws \Badoo\Jira\Exception\Issue
+     * @throws \Badoo\Jira\REST\Exception
+     */
+    public function getIssue(string $issue_key) : \Badoo\Jira\Issue
+    {
+        return \Badoo\Jira\Issue::byKey($issue_key, [], [], $this->RESTClient);
+    }
+
+    /**
+     * @param string ...$keys
+     *
+     * @return array
+     *
+     * @throws \Badoo\Jira\Exception\Issue
+     * @throws \Badoo\Jira\REST\Exception
+     *
+     * @see \Badoo\Jira\Issue::ByKeys
+     */
+    public function getIssues(string ...$keys) : array
+    {
+        return \Badoo\Jira\Issue::byKeys($keys);
+    }
+
+    /**
+     * @param string $jql - search query string. {@link https://deviniti.com/atlassian/a-short-guide-to-the-jira-query-language/ Short introduction}
+     * @param int $limit
+     * @param int $offset
+     *
+     * @return array
+     *
+     * @throws \Badoo\Jira\Exception\Issue
+     * @throws \Badoo\Jira\REST\Exception
+     *
+     * @see \Badoo\Jira\Issue::search
+     */
+    public function searchIssue(string $jql, int $limit, int $offset) : array
+    {
+        return \Badoo\Jira\Issue::search($jql, [], [], $limit, $offset, $this->RESTClient);
+    }
+
+    /**
+     * @param string $project_key - key of project for new issue (e.g. EX, TEST, IOS)
+     * @param string|int $issue_type - textual name or ID of type for issue you are going to create (e.g. 'Bug' or 34)
+     *
+     * @return \Badoo\Jira\Issue\CreateRequest
+     *
+     * @throws \Badoo\Jira\Exception\Issue
+     * @throws \Badoo\Jira\REST\Exception
+     *
+     */
+    public function createIssue(string $project_key, $issue_type) : \Badoo\Jira\Issue\CreateRequest
+    {
+        return new \Badoo\Jira\Issue\CreateRequest($project_key, $issue_type, $this->RESTClient);
+    }
+
+    /**
+     * Delete issue from Jira.
+     *
+     * **DANGER**: This action is applied immediately and can't be undone. Be careful.
+     *
+     * @param string $issue_key
+     * 
+     * @throws \Badoo\Jira\Exception\Issue
+     * @throws \Badoo\Jira\REST\Exception
+     */
+    public function deleteIssue(string $issue_key) : void 
+    {
+        $this->getIssue($issue_key)->delete();;
+    }
+
+    /**
+     * Get user from API by ID (lazy load)
+     *
+     * @param string $name - name of user in JIRA.
+     *
+     * <b>Don't mess with display name you see in UI!</b>
+     *
+     * @return \Badoo\Jira\User
+     */
+    public function getUser(string $name) : \Badoo\Jira\User
+    {
+        return new \Badoo\Jira\User($name, $this->RESTClient);
+    }
+
+    /**
+     * Search for user by exact match in email address
+     *
+     * @param string $email - user email
```

Comment:
Do we really need the minus character between variable name and its description?